### PR TITLE
update regions in readme

### PR DESCRIPTION
**Pull request by jrknezha**
In the readme, the regions table was not updated to reflect the changes last summer. This is just making the readme accurate to the regions table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,7 +401,10 @@ regions
 
     id = Column(Integer, primary_key=True, autoincrement=True)
     name = Column(String(79), nullable=False)
-    bounds = Column(String(255), nullable=False)
+    min_lat = Column(Float, nullable=False)
```
Comment (amschne) [inline README.md]:
```suggestion
    min_lat = Column(Float, nullable=False) # degrees north of equator [-90, 90)
```

File: README.md [hunk 2]
```diff
@@ -401,7 +401,10 @@ regions
...
+    max_lat = Column(Float, nullable=False)
```
Comment (amschne) [inline README.md]:
```suggestion
    max_lat = Column(Float, nullable=False) # degrees north of equator (-90, 90]
```

File: README.md [hunk 3]
```diff
@@ -401,7 +401,10 @@ regions
...
+    east_lon = Column(Float, nullable=False)
```
Comment (amschne) [inline README.md]:
```suggestion
    east_lon = Column(Float, nullable=False) # degrees east of prime meridian [0, 360)
```

File: README.md [hunk 4]
```diff
@@ -401,7 +401,10 @@ regions
...
+    west_lon = Column(Float, nullable=False)
```
Comment (amschne) [inline README.md]:
```suggestion
    west_lon = Column(Float, nullable=False) # degrees east of prime meridian (0, 360]
```

**Review by amschne — APPROVED** [review]
Nice catch - thanks for updating. I suggest adding comments regarding units and bounds (see below), for clarity on the expected values.

**Comment by jrknezha** [comment]
@amschne great suggestion, I've confirmed the suggestions into branch

**Pull request merged**